flink/src/main/java/org/apache/iceberg/flink/IcebergTableSource.java

```diff
@@ -112,6 +119,11 @@ public String explainSource() {
       explain += String.format(", LimitPushDown : %d", limit);
     }
 
+    if (filters != null) {
+      explain += String.format(", FilterPushDown,the filters :%s",
+          filters.stream().map(filter -> filter.toString()).collect(Collectors.joining(",")));
```

Comment:
I think returning `null` in the filters class will cause NPE here as well

Reply:
nit:  it's more simple to rewrite this as: 

```java
 explain += String.format(", FilterPushDown,the filters :%s", Joiner.on(",").join(filters));
```